tests/scanner/scanners/fw_rules_scanner_test.py

```diff
@@ -0,0 +1,437 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Scanner runner script test."""
+
+from datetime import datetime
+import mock
+import unittest
+import parameterized
+
+from tests.unittest_utils import get_datafile_path
```

Comment:
I would prefer you import unittest_utils and then call unittest_utils.get_datafile_path and unittest_utils.ForsetiTestCase.

Reply:
Done.